autopush/router/apns2.py

```diff
@@ -21,6 +22,16 @@
 APNS_PRIORITY_LOW = '5'
 
 
+class ComplexEncoder(json.JSONEncoder):
+    def default(self, obj):
+        if isinstance(obj, Decimal):
+            return int(float(obj.to_integral()))
```

Comment:
Why not use to_integral_value? The docs indicate it should already return an int.